addons/im_livechat/static/src/legacy/models/public_livechat_message.js

```diff
@@ -73,8 +73,10 @@ const PublicLivechatMessage = Class.extend({
      */
     getAvatarSource() {
         let source = this._serverURL;
-        if (this.hasAuthor()) {
+        if (this.isOperatorTheAuthor()) {
             source += `/im_livechat/operator/${this.getAuthorID()}/avatar`;
+        } else if (this.hasAuthor() && session.user_id) {
```

Comment:
Should use `isVisitorTheAuthor`
`_isMyselfAuthor` is not working properly, but it's being handled by Task-3010650

So this won't be fixed immediately, but at least it will rely on the proper method, and it will be fully fixed with Task-3010650.

We might wait for Task-3010650 to be merged before merging this one.